src/main/java/server/Server.java

```diff
@@ -1,21 +1,35 @@
 package main.java.server;
 
+
+import lombok.extern.java.Log;
+import main.java.message.Request;
+
+import java.io.BufferedReader;
 import java.io.IOException;
+import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.net.ServerSocket;
 import java.net.Socket;
 
-
+@Log
 public class Server {
 
     private static final int PORT = 8080;
 
     public static void main(String[] args) {
-        try (Socket socket  = new ServerSocket(PORT).accept()) {
-            OutputStream stream = socket.getOutputStream();
-            stream.write("helloWorld".getBytes());
+
+        connetClient();
+    }
+
+    private static void connetClient() {
+        try (Socket socket = new ServerSocket(PORT).accept()) {
+            OutputStream out = socket.getOutputStream();
+            BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            Request request = Request.init(in);
+            System.out.println(request.toString());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=4&issues=AWQvx9Pe_bEzQ-kUjrAu&open=AWQvx9Pe_bEzQ-kUjrAu)

[ISSUE_KEY]: AWQvx9Pe_bEzQ-kUjrAu